packaging/windows/abacus.iss

```diff
@@ -1,7 +1,16 @@
-; InnoSetup 6.6
+; Built with InnoSetup 6.6, make sure to use this version
+; Using a different version of InnoSetup can give different results
+
+#define GetLatestTag() \
+  Local[0] = "/S /C pushd """ + SourcePath + """ && git.exe describe --tags --abbrev=0 > version.txt && popd", \
```

Comment:
This command returns the most recent tag even if there have been commits after that. We can avoid this by also passing `--exact-match`, but that makes the command fail if the current commit is not tagged.

Maybe @rnijveld has some good idea on how to tackle this, since he is working on #2516.